docs/docs/mapillary.config.api/mapillary.config.api.entities.md

```diff
@@ -143,19 +143,23 @@ Fields:
 8. computed_compass_angle - float, compass angle after running image processing
 9. computed_geometry - GeoJSON Point, location after running image processing
 10. computed_rotation - enum, corrected orientation of the image
-11. exif_orientation - enum, orientation of the camera as given by the exif tag
+11. creator - the username and user ID who owns and uploaded the image
+12. exif_orientation - enum, orientation of the camera as given by the exif tag
     (see: https://sylvana.net/jpegcrop/exif_orientation.html)
-12. geometry - GeoJSON Point geometry
-13. height - int, height of the original image uploaded
-14. thumb_256_url - string, URL to the 256px wide thumbnail
-15. thumb_1024_url - string, URL to the 1024px wide thumbnail
-16. thumb_2048_url - string, URL to the 2048px wide thumbnail
-17. merge_cc - int, id of the connected component of images that were aligned together
-18. mesh - { id: string, url: string } - URL to the mesh
-19. quality_score - float, how good the image is (experimental)
-20. sequence - string, ID of the sequence
-21. sfm_cluster - { id: string, url: string } - URL to the point cloud
-22. width - int, width of the original image uploaded
+13. geometry - GeoJSON Point geometry
+14. height - int, height of the original image uploaded
+15. make - string, the manufacturer name of the camera device
+16. model - string, the model or product series name of the camera device
+17. thumb_256_url - string, URL to the 256px wide thumbnail
+18. thumb_1024_url - string, URL to the 1024px wide thumbnail
+19. thumb_2048_url - string, URL to the 2048px wide thumbnail
+20. thumb_original_url - string, URL to the original wide thumbnail
+21. merge_cc - int, id of the connected component of images that were aligned together
+22. mesh - { id: string, url: string } - URL to the mesh
+23. quality_score - float, how good the image is (experimental)
+24. sequence - string, ID of the sequence
+25. sfm_cluster - { id: string, url: string } - URL to the point cloud
+26. width - int, width of the original image uploaded
 ```
```

Comment:
I think there needs to be the mention for detections as well, given the last item under `width` for the fields under the `Image` entity,

https://www.mapillary.com/developer/api-documentation#image

<img width="828" alt="image" src="https://github.com/mapillary/mapillary-python-sdk/assets/41635766/55d7ed4d-8dd4-451a-9ed1-88153e8dadda">